udata/search/__init__.py

```diff
@@ -1,13 +1,11 @@
 import logging
+import requests
 
 from flask import current_app
 from mongoengine.signals import post_save, post_delete
-from udata_event_service.producer import produce
 
-from udata.models import db, Dataset, Organization, Reuse
+from udata.models import db
 from udata.tasks import task, as_task_param
-from udata.event.values import KafkaMessageType
```

Comment:
Since we don't import from udata.event here anymore, udata.core.dataset.events is never imported when starting app, thus not connecting signals. It doesn't fail in test, since it is explicitly imported.